src/pcp2openmetrics/pcp2openmetrics.py

```diff
@@ -483,14 +486,24 @@ def openmetrics_labels(inst, name, desc, labels):
 
         body = ''
         for metric in results:
+            #variable declaration
             i = list(self.metrics.keys()).index(metric)
             desc = self.pmconfig.descs[i]
             context = self.pmfg.get_context()
             pmid = context.pmLookupName(metric)
             labels = context.pmLookupLabels(pmid[0])
+            semantics = self.context.pmSemStr(desc.contents.sem)
+            units = desc.contents.units
+            pmIDStr = context.pmIDStr(pmid[0])
+            pmIndomStr = context.pmInDomStr(desc)
+            
             help_dict = {}
             help_dict[metric] = context.pmLookupText(pmid[0])
 
+            if self.context.type == PM_CONTEXT_ARCHIVE:
+                body += "\nMetric %s details (last fetch: %d)\n:" % (metric, ts)
```

Comment:
@lmchilton this bit doesn't look right...?

Reply:
@natoscott that is the part that adds the timestamps in archive mode. What part does not look right?  The body string format I took from pcp2template, if it should be in a different format?

Reply:
(discussed elsewhere, but yep, different format needed)